tests/event-bus/e2e-tester/e2e-tester.go

```diff
@@ -28,6 +28,7 @@ import (
 const (
 	eventType1              = "test-e2e-1"
 	eventType2              = "test-e2e-2"
+	eventHeaderType         = "test-e2e-header-2"
```

Comment:
Kindly keep it consistent, either remove the `-2` from the value or add it to the var name

Reply:
Removed `-2`